### PR TITLE
SW-1568 Extract snackbar as an easier to use utility

### PR DESCRIPTION
**Pull request by karthikbtf**
- added some replaced examples to illustrate
- we can replace the other use-cases in smaller chunks

### CODE REVIEW

File: src/utils/useSnackbar.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  // page helpers
+  const pageInfo = (msg: Message, title?: Title, closeCallback?: OnCloseCallback) =>
+    page(msg, title, 'info', closeCallback);
+  const pageWarning = (msg: Message, title?: Title, closeCallback?: OnCloseCallback) =>
+    page(msg, title, 'warning', closeCallback);
+  const pageSuccess = (msg: Message, title?: Title, closeCallback?: OnCloseCallback) =>
+    page(msg, title, 'success', closeCallback);
+  const pageError = (msg: Message, title?: Title, closeCallback?: OnCloseCallback) =>
+    page(msg, title, 'critical', closeCallback);
```
Comment (karthikbtf) [inline src/utils/useSnackbar.ts:41]: wanted to use typed props here but figured this is easier to use in most of the use-cases

File: src/components/EditOrganization/index.tsx [hunk 2]
```diff
@@ -113,18 +112,10 @@ export default function OrganizationView({ organization, reloadOrganizationData
     } else {
       const response = await updateOrganization(organizationRecord);
       if (response.requestSucceeded) {
-        setSnackbar({
-          type: 'toast',
-          priority: 'success',
-          msg: 'Changes saved',
-        });
+        snackbar.toastSuccess('Changes saved');
```
Comment (constanzauanini) [inline src/components/EditOrganization/index.tsx:115]: This should have use string files, but I know it was like that before haha. So no need to change it now

Comment (karthikbtf) [reply src/components/EditOrganization/index.tsx:115]: yeah, i saw that. i will move it to strings.

File: src/utils/useSnackbar.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      onCloseCallback,
+      priority,
+    });
+  };
+
+  // toast helpers
+  const toastInfo = (msg: Message, title?: Title) => toast(msg, title, 'info');
+  const toastWarning = (msg: Message, title?: Title) => toast(msg, title, 'warning');
+  const toastSuccess = (msg: Message, title?: Title) => toast(msg, title, 'success');
+  const toastError = (msg?: Message, title?: Title) => toast(msg || strings.GENERIC_ERROR, title, 'critical');
```
Comment (constanzauanini) [inline src/utils/useSnackbar.ts:31]: I like this default of generic error :)

**Review by constanzauanini — APPROVED** [review]

**Comment by karthikbtf** [comment]
Thanks @constanzauanini i will update usages of other places in follow up PRs.

**Pull request merged**